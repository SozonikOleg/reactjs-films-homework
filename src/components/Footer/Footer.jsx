import React from 'react';
import styles from './Footer.module.scss';

const Footer = () => (
  <div className={styles.footer_list}>
    <p className={styles.footer_header}>Films</p>
    <span className={styles.footer_description}>Copyright &copy; 2018 FILMS YOUR NAME</span>
  </div>
);

export default Footer;