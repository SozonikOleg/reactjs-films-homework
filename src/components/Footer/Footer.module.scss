.footer_list {
    text-align: center;
}

.footer_header {
    margin-top: 52px;
    font-size: 42px;
    text-transform: uppercase;
    color: #09ade3;
}

.footer_description {
    margin-top: 46px;
    display: block;
    font-size: 22px;
    color: #c5c5c5;
}