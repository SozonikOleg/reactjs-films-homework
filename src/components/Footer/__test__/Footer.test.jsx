import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Footer from '../Footer';

it('Footer renders well without props', () => {
  const renderer = new ShallowRenderer();
  const result = renderer.render(<Footer />);
  expect(result).toMatchSnapshot();
});
