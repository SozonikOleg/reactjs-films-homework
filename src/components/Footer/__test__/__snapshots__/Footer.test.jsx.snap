// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders well without props 1`] = `
<div
  className="footer_list"
>
  <p
    className="footer_header"
  >
    Films
  </p>
  <span
    className="footer_description"
  >
    Copyright Â© 2018 FILMS YOUR NAME
  </span>
</div>
`;
